docker/Dockerfile.psga-pipeline-sars-cov-2

```diff
@@ -46,9 +42,7 @@ COPY data/sars_cov_2/primer_schemes /primer_schemes
 # READ IT AND KEEP
 COPY data/sars_cov_2/read-it-and-keep/* /
 
-WORKDIR /app
-
-ENV PYTHONPATH=/app/
+ENV PYTHONPATH=/
```

Comment:
As per my comment in the CI workflow, is this worth a comment here explaining why this is being set?